jupyter-book/conditions/perturbation_modeling.ipynb

```diff
@@ -22,12 +22,12 @@
    "metadata": {},
```

Comment:
<p>I think this is a superb example of where the method is not described but simply shown in the tutorial, and if someone wants to look at how it works exactly they can look it up but otherwise the tutorial is kept clean.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/cross-modal-single-cell-best-practices/pull/91/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='137'/>